packages/collector/test/tracing/cloud/aws/sqs/sqs-consumer.js

```diff
@@ -36,10 +37,18 @@ const consumerApp = Consumer.create({
   queueUrl: queueURL,
   sqs,
   handleMessage: async message => {
+    const span = instana.currentSpan();
+    span.annotate('sdk.custom.tags.my_tag', 'group1');
```

Comment:
AFAICT these two lines serve no purpose in the integration test and were only needed for a manual experiment. Maybe remove?